src/content/developers/tutorials/eip-1271-smart-contract-signatures/index.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "EIP-1271: Signing and Verifying Smart Contract Signatures"
+description: An overview of smart contract signature generation and verification with EIP-1271. We also walk through Gnosis Safe's EIP-1271 implementation to provide a concrete example for smart contract developers to build on.
+author: nathanhleung
+lang: en
+tags: ["eip-1271", "smart contracts", "verifying", "signing"]
+skill: intermediate
+published: 2023-01-12
+---
+
+The [EIP-1271](https://eips.ethereum.org/EIPS/eip-1271) standard allows smart contracts to verify signatures.
+
+In this tutorial, we give an overview of digital signatures, EIP-1271's background, and [Gnosis Safe](https://gnosis-safe.io/)'s specific implementation of EIP-1271. All together, this can serve as a starting point for implementing EIP-1271 in your own contracts.
```

Comment:
```suggestion
In this tutorial, we give an overview of digital signatures, EIP-1271's background, and [Safe](https://app.safe.global/)'s specific implementation of EIP-1271. All together, this can serve as a starting point for implementing EIP-1271 in your own contracts.
```

I believe they rebranded now from Gnosis Safe to Safe. Just updating name + url.

Reply:
Ah yes I think you're right — I can go thru and make this change throughout

Reply:
Might still be useful to add a parenthetical (previously Gnosis Safe) like they do on their website

<img width="662" alt="Screenshot 2023-03-01 at 13 34 56" src="https://user-images.githubusercontent.com/10874225/222270044-9a34ac6f-255d-4cd9-8e15-fd3bf55a65f7.png">